### DISCUSSION TITLE
Is there any way we can override the specific configuration of a board on an undetected platform

### DISCUSSION THREAD

**Discussion by thuvasooriya** [adafruit/Adafruit_Blinka#805] [Q&A] [answered]
This is very convenient for testing and to monkeypatch some configs on the fly. 
Say if we can make the Blinka library use a configuration of a specific board in the current SBC without failing the platform detect because the current platform is not available. I assume we can do this by passing some arguments or something along somewhere. Excuse that fact that I'm a newbie and please let me know if there is such an implementation or how should I go about this.

My goal is to make the Blinka library work on my Orange Pi Zero 2W which has an Alwinner H618 which is very similar to H616

**Comment by makermelissa** [comment, marked as answer]
Yes. I had added this to PlatformDetect a while back and while reviewing https://github.com/adafruit/Adafruit_Python_PlatformDetect/pull/347 I had remembered it. If you set the BLINKA_FORCE_CHIP and BLINKA_FORCE_BOARD environment variables to the board constants, you can have it use those boards. See the PR I linked for more details.
